CONTRIBUTING.md

```diff
@@ -30,6 +30,13 @@ Dependencies that are added to the `package.json` file at the root of the
 repo are shared between packages. Dependencies listed in the
 `package.json` files within each individual package directory work as normal.
 
+### For Windows users
+There is a known [bug](https://github.com/lerna/lerna/issues/385#issuecomment-948402686) which running `lerna bootsrap` fails to generate the `pwa-kit-cli` bin shims
+in the bin file on Windows.
+As a result, some npm scripts that uses `pwa-kit-cli` commands (e.g `npm start` in `template-retail-react-app`) will run on Windows Host Script instead of Node, which will cause the script to fail
+
+To fix this issue, please run `npm run lerna link --force-local` after `npm ci` finishes running
```

Comment:
Dumb question.. but could we not do this step for the users in our bootstrap script? 


Something like:

```
if (process.platform === 'win32') {
    childProc.execSync('npm run lerna link --force-local', {stdio: 'inherit'})
}
```

Reply:
I tested it out on the Windows VM. It works. I will update the script instead of readme. Thanks for the suggestion!

Reply:
@bendvc I've made the change. Please have another look if you get a chance. Thanks.